typings/index.d.ts

```diff
@@ -2577,6 +2574,19 @@ declare module 'discord.js' {
     name?: string;
   }
 
+  interface GuildCreateOptions {
+    afkChannelID?: number;
+    afkTimeout?: number;
+    channels?: PartialChannelData[];
+    defaultMessageNotifications?: DefaultMessageNotifications | number;
+    explicitContentFilter?: ExplicitContentFilterLevel | number;
+    icon?: BufferResolvable | Base64Resolvable | null;
+    region?: string;
+    roles?: PartialRoleData[];
+    systemChannelID?: number;
```

Comment:
```suggestion
    systemChannelID?: Snowflake;
```
The type is snowflake in Discord's doc.